File: Module 2/27_Fibonacci_Factorial/Python/Fibonacci.py
def fibonacci(n):
    if n == 0:
        return 0
    elif n == 1:
        return 1
    else:
        return fibonacci(n - 1) + fibonacci(n - 2)

if __name__ == "__main__":
    n = int(input("Please Enter the Position Number: "))
    print(f"Fibonacci Number at Position {n}: {fibonacci(n)}")

File: Module 2/27_Fibonacci_Factorial/Java/Fibonacci.java
import java.util.Scanner;

public class Fibonacci {
    public static int fibonacci(int n) {
        if (n == 0)
            return 0;
        else if (n == 1)
            return 1;
        else
            return fibonacci(n - 1) + fibonacci(n - 2);
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Please Enter the Position Number: ");
        int n = sc.nextInt();
        System.out.println("Fibonacci Number at Position " + n + ": " + fibonacci(n));
    }
}
